README.md

```
@@ -425,7 +408,7 @@ This makes the visualization more balanced and customizable for different types
   * `overlay.*`: Renders the ImGui debug overlay.
   * `logging.*`: Simple thread-safe logging.
   * `listeningway_addon.cpp`: Main addon entry point, event handling, initialization.
-  * `settings.*`: Loads/saves settings from `.ini`, holds the `ListeningwaySettings` struct.
+  * `settings.*`: Loads/saves settings from `.json`, holds the `ListeningwaySettings` struct.
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 0; Actual: 2

(MD007, ul-indent)